roles/system_info/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+# tasks file for system_info
+- name: Get system info
+  ansible.builtin.command: '/usr/bin/rpm-ostree status --json'
+  changed_when: false
+  register: system_info_output
+
+- name: Set rpm-ostree facts on system
+  vars:
+    system_info_json: "{{ system_info_output.stdout | from_json }}"
+  ansible.builtin.set_fact:
+    cacheable: true # noqa var-naming[no-role-prefix]
+    system_info_rpm_ostree:
```

Comment:
Not trying to nitpick, but I'd suggest this be `rpm_ostree` instead of `system_info_rpm_ostree`, purely because the `ansible_facts` dict is a known quantity to contain system information so that feels to me like extra typing for not much gain.

Reply:
I agree buuuut lint decided that's the way so I had to change it. The new lint standards are internal vars need to be prefixed by the role name. 

I can add the flag `no qa var-naming[no-role-prefix]` like I did with `cacheable` because lint thought that was a var for `set_fact`